pkg/engine/resolve/resolve_test.go

```diff
@@ -2243,7 +2243,7 @@ func TestResolver_ResolveGraphQLResponse(t *testing.T) {
 									},
 									{
 										SegmentType: StaticSegmentType,
-										Data:        []byte(`,"firstArg":`),
+										Data:        []byte(`,"firstArg":"`),
```

Comment:
I didn't understand why we need an extra `"` character here.

Reply:
It's closed on line 2256

Reply:
I got it. Thank you, David.